src/v/cluster/controller_backend.cc

```diff
@@ -300,6 +301,10 @@ void controller_backend::setup_metrics() {
 ss::future<> controller_backend::start() {
     setup_metrics();
     return bootstrap_controller_backend().then([this] {
+        if (ss::this_shard_id() == ss::shard_id{0}) {
+            ssx::spawn_with_gate(
+              _gate, [this] { return clear_orphan_topic_files(); });
+        }
```

Comment:
`clear_topic_orphan_files` can throw `std::filesystem::filesystem_error`. It would be nice to log something in that case. Also, if I recall correctly, seastar doesn't like background failed futures.

Reply:
fixed